pkg/manifests/manifests_test.go

```diff
@@ -1194,6 +1194,74 @@ ingress:
 	}
 }
 
+func TestPrometheusQueryLogFileConfig(t *testing.T) {
+	testCases := []struct {
+		name             string
+		queryLogFilePath string
+		expected         string
+		errString        string
+	}{
+		{
+			name:             "basic config",
+			queryLogFilePath: "/tmp/query.log",
+			expected:         "/tmp/query.log",
+			errString:        "",
+		},
+		{
+			name:             "basic config",
+			queryLogFilePath: "/dev/query.log",
```

Comment:
I'm not sure that Prometheus would start successfully in this case :thinking:

Reply:
I'm also not sure, I'm spinning up a cluster-bot to test. If it's doesn't we can add that to the tests and allow "/dev/stdout" but forbid anything other than know outputs: /dev/null /dev/stdout